app/jobs/add_inbox_tag_to_thread_job.rb

```diff
@@ -0,0 +1,11 @@
+class AddInboxTagToThreadJob < ApplicationJob
+  def perform(message_thread)
+    message_thread.assign_tag(message_thread.tenant.inbox_tag) if message_thread.messages.any?{ |message| significant_inbox_message?(message) }
+  end
+
+  private
+
+  def significant_inbox_message?(message)
+    !message.outbox? && !Govbox::Message::INSIGNIFICANT_MESSAGE_CLASSES.include?(message.metadata.dig('edesk_class'))
```

Comment:
Tu nam leakuje domena upvs a trosku sa mi zda, ze uz si tu tahame a biznisovu domenu, ved "nepodstatne spravy" uz mame - to su tie co sa default schovavaju. Nedava zmysel proste tu spravu oznacit ako schovanu uz pri vstupe a vyriesi nam to aj tento problem?

Reply:
Ano, tak to aj robime, standardne sa tag pridava pri spracovani novej spravy, https://github.com/slovensko-digital/govbox-pro/blob/GO-181/filter_only_delivered_messages/app/models/govbox/message.rb#L42
Ale potrebujeme nejako spatne updateovat uz existujuce spravy, vid https://github.com/slovensko-digital/govbox-pro/pull/546#discussion_r1943176701

Reply:
A nevies vyuzit tu metodu `.insignificant?` (inak negativne nazvy nie su uplne dobre, pozitivne `significant?` davaju vacsi zmysel, kvoli podmienkam.

Reply:
Aktualne nie, lebo to je metoda nad `Govbox::Message`, tu uz pracujeme s `Message`.
Ale tento job naozaj raz zbehne a mazeme ho zo zdrojakov, viac ho nebudeme potrebovat. Negativny nazov mozem zmenit